JustTrack/JEEventsGenerator/main.swift

```diff
@@ -348,7 +358,7 @@ do {
     //write struct file
     let structSwiftFilePath = CommandLine.arguments[2]
     if !FileManager.default.fileExists(atPath: structSwiftFilePath) {
-        throw JEStructsGeneratorError.swiftFileNotFound
+        FileManager.default.createFile(atPath: structSwiftFilePath, contents: .none, attributes: .none)
```

Comment:
Actually, we want to fail if the .swift file isn't there, let's imagine these scenarios if the file doesn't exist:

1. We silently create and fill the .swift file, that isn't added to the project/target, so the build is completed without error but developer can't use JustTrack and the generated events.
2. We fail the build and display clearly the failures reasons, the developer can amend the path or create the file.

we definitely prefer the 2nd scenario.